learning_journal/models/mymodel.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import (
+    Column,
+    Index,
+    Integer,
+    Text,
+)
+
```

Comment:
Just make sure that when you do, you also remove the line in `models/__init__.py` where this model is imported.